app/src/main/java/com/zeapo/pwdstore/SearchableRepositoryViewModel.kt

```diff
@@ -254,8 +254,9 @@ class SearchableRepositoryViewModel(application: Application) : AndroidViewModel
         }.asLiveData(Dispatchers.IO)
 
     private fun shouldTake(file: File) = with(file) {
+        if (showHiddenContents) return true
```

Comment:
It seems to me that we'll end up showing non-hidden non-gpg files this was too. Is that intended?

Tests look good.

Reply:
Yep, the toggle's now essentially a boolean choice between 'Show everything' and 'Show generally useful things'. I've retained the previous wording since it seems to carry over to users a bit better in my head, but I'm willing to switch it to just say 'Show all files and folders'.

Reply:
I would personally prefer to switch to "Show all files and folders". I'm not sure whether every user knows that dotfiles are "hidden", but if they don't see a file they expect to be there, they would likely give a preference with the name "Show all ..." a try.